### DISCUSSION TITLE
How can we get the difference between the node features based on the edge index in a graph classification task?

### DISCUSSION THREAD

**Discussion by arake001** [pyg-team/pytorch_geometric#3952] [Q&A]
Suppose we have 10 nodes in a graph. Based on the edge_index, I want the difference between the node features attributes.

Something like sub_node_features = w(hpt - hjt). Here **p** node is the neighboring node for node **j** from edge index. **t** represents the number of node feature attributes.

**Comment by rusty1s** [comment]
Do you mean something like:
```python
out = x[edge_index[0]] - x[edge_index[1]]
```
